DependencyInjection/Configuration.php

```diff
@@ -85,6 +85,9 @@ public function getConfigTreeBuilder()
                         ->scalarNode('enable_sibling_provider')->defaultValue(false)->end()
                     ->end()
                 ->end()
+                ->arrayNode('content_listener')
+                    ->canBeDisabled()
```

Comment:
does the bundle need to know the service id of the default listener? if yes that would be the right thing to do. although, would that mean that if its the default value, SeoBundle loads the service and otherwise its assumes the application provides the service somewhere else? and, does the SeoBundle need to know the name of that listener, does it inject it elsewhere?